octoprint_mrbeam/static/js/app/view-models/design-store.js

```diff
@@ -11,23 +11,31 @@ $(function () {
         // 'http://localhost:8080';
         self.DESIGN_STORE_IFRAME_HEALTHCHECK_SRC =
             self.DESIGN_STORE_IFRAME_SRC + "/api/healthcheck";
+        self.DESIGN_STORE_TAB_ELEMENT = $("#designstore_tab_btn");
 
         self.loginState = params[0];
         self.navigation = params[1];
         self.analytics = params[2];
         self.settings = params[3];
+        self.laserheadChangedVM = params[4];
 
         self.lastUploadedDate = ko.observable("");
         self.eventListenerAdded = ko.observable(false);
 
+        self.onUserLoggedIn = function () {
```

Comment:
when is this being called?

Reply:
When the user logs in or after page loads and the user is already logged in